### ISSUE TITLE
Диплом

### ISSUE THREAD

**Issue by timyrka67** [timyrka67/timyrka67.github.io#1]
@aurusov

[РПЗ](http://timyrka67.github.io/Diploma/diploma.pdf)
[Презентация](https://docs.google.com/presentation/d/10VFJoRkrkd8fQpT_MQ2BKZe1_xv-rv6XCw0Zp4AxYmg/edit?usp=sharing)
[Рецензия](http://timyrka67.github.io/Diploma/fb.pdf)
[Аннотация](http://timyrka67.github.io/Diploma/a.pdf) 
[Диаграмма классов симулятора](http://timyrka67.github.io/Diploma/1.pdf)
[Диаграмма классов проигрывателя](http://timyrka67.github.io/Diploma/2.pdf)

**Comment by aurusov** [comment]
Листы
Диаграмма классов симулятора
- В штампе используются разные шрифты или разные стили одного шрифта. Режет глаз.
- Зачем тебе эта диаграмма ? Какие класс твои ?
- Лист нужен в pdf

Диаграмма классов проигрывателя
- В штампе используются разные шрифты или разные стили одного шрифта. Режет глаз.
- Много белого. Отцентрируй по высоте с учетом рамки, а не как сейчас - без неё.
- Лист нужен в pdf

Записка
- Выводы
  - `описано понятие состояния модели<где запятая ?> разработаны`

Титлы, ok, **[на печать]**
Рецензия, ok, **[на печать]**
Аннотация, ok, **[на печать]**

Презентация
- Пока не смотрел. Жду готовности, хотя бы какой-то части.

**Comment by aurusov** [comment]
Почитал, отписался.

**Comment by aurusov** [comment]
Отписывайся о прогрессе в этой таске.

**Comment by aurusov** [comment]
Где титлы ? Где презентация ?

**Comment by timyrka67** [comment]
[Титлы добавил](http://timyrka67.github.io/Diploma/dp.pdf) -научно-исследовательская часть написана так сознательно. Я просто заполнял PDF.
Презентация пока не закончена

**Comment by aurusov** [comment]
> Презентация пока не закончена

Но ссылку-то можно разместить. Ну что же такое. Тебя прям приятно, что я трачу кучу времени на поиск этой таски, на поиск ссылок на титлы или презентацию ? Если нет, то почему замечание

> Надо выложить на гитхаб все материалы.

уже 5 дней без ответа. И только со ВТОРОГО напоминания выложил титлы. Теперь ТРЕТИЙ раз - выложи презентацию !!! Не знаю как по другому говорить.

**Comment by aurusov** [comment]
Посмотрел. Можно править записку. Если уже начал это делать, то перечитай все замечания заново, я их только что закончил править.

**Comment by timyrka67** [comment]
Записка

> [12]. рис. 2.6.3 - когда появится нормальный ?

Не появится, он и так нормальный. В презентации на слайде 9 есть вариант со штриховкой. 
В презентации менять не буду ничего. В записке вставлю со слайда 9. 

> [17]. Разный стиль подписей к рисункам. То по центру, то справа, то по центру, но наезжает на рисунок. Все должно быть сделано через стили ворда, с автоматической нумерацией.:question:

Поправил. Через стили ворда не так уж и хорошо. Выравнивание будет нормально, но если картинку сдвинуть все уедет. Поэтому я делают вставку надписи и превязываю ее к картинке. 

> [21]. `V` - это не совсем скорость, хотя бы из-за её размерности. Кстати, какая у неё размерность ?

Ответ:[масштаб]/[секунду] т.е. безразмерная величина 

> [22]. Что за трешь с названиями. `M - величина масштаба единиц симулятора`. Это про что ? На рис.2.6.3, который называется ФОРМУЛА, Карл, эта величина называется `deltaSim * Scale`. Тебе придется дать всем этим переменным читаемые и названия и обозначения в формулах.

Сейчас сделал так:
V -> playSpeed
V\* -> k
M оставил.  

> [23]. Радом с каждой формулой должен быть её номер

В ворде я этого делать не буду. Эта адовая система  с неадекватным форматированием отнимает у меня жизнь. Перед печатью я пронумерую все в pdf. В pdf 5 минут делать.

Титлы

> У меня поехали надписи
> Открыл в другом вьювере - стало лучше, хотя и странно. Не должен пдф себя так вести. Где гарантия, что ты так же распечатаешь ?

Наверное буду внимательно проверять, что напечатаю или как-нибудь переведу в растр
.

> Не понял, что надо посмотреть кроме темы. Ведь ничего не заполнено.

Будет только рукописная версия. Все что не заполнина буду писать руками.

Презентация

Практически не менял

Обновил записку.

**Comment by aurusov** [comment]
> Не появится, он и так нормальный. В презентации на слайде 9 есть вариант со штриховкой.
> В презентации менять не буду ничего. В записке вставлю со слайда 9. 

Не согласен. Надо изменить на тот, который был на бумаге с консультации. Говорим про это уже не в первый раз. Твоя категоричность напорится на мою. Или приведи аргументы.

**Comment by timyrka67** [comment]
> Не согласен. Надо изменить на тот, который был на бумаге с консультации. Говорим про это уже не в первый раз. Твоя категоричность напорится на мою. Или приведи аргументы.

Вот так?

![snczetr6jkm](https://cloud.githubusercontent.com/assets/8823619/16178426/00c9e5b4-3650-11e6-9114-26acc6cdcc1c.jpg)

**Comment by aurusov** [comment]
Да. Тут отмечено _пять_ областей. У тебя только _три_. Как они будут отмечены - штрихом или дугами, как у тебя - не важно. Важно их выделить и подписать, хоть схематично.

**Comment by aurusov** [comment]
> Ответ:[масштаб]/[секунду] т.е. безразмерная величина 

Хм... Ну как бы сама запись [масштаб]/[сек] != безразмерной величине. Только у нас не [масштаб]/[сек], а просто масштаб: ([сек]/[сек]) \* koef. Т.е. безразмерная величина, но по другим причинам. Видимо это место надо обсудить голосом.

**Comment by aurusov** [comment]
> В ворде я этого делать не буду. Эта адовая система с неадекватным форматированием отнимает у меня жизнь. Перед печатью я пронумерую все в pdf. В pdf 5 минут делать.

https://www.google.ru/search?q=%D0%BD%D1%83%D0%BC%D0%B5%D1%80%D0%B0%D1%86%D0%B8%D1%8F+%D1%84%D0%BE%D1%80%D0%BC%D1%83%D0%BB+%D0%B2+%D0%B2%D0%BE%D1%80%D0%B4%D0%B5&ie=utf-8&oe=utf-8&gws_rd=cr&ei=fMZmV5_zN4f_sQGzjYrYBQ

**Comment by aurusov** [comment]
Посмотрел

**Comment by aurusov** [comment]
Ты чего не перезваниваешь ? До 22:00 готов по скайпу или телефону.

**Comment by timyrka67** [comment]
![default](https://cloud.githubusercontent.com/assets/8823619/16179487/05604760-3670-11e6-9ca4-e14177b1d566.jpg)

**Comment by aurusov** [comment]
- нет названия у М. если это масштаб, то он не там подписан
- что такое [1] после М ?
- у k не должно быть размерности
- что такое [1] после deltaTReal ?

**Comment by timyrka67** [comment]
![16](https://cloud.githubusercontent.com/assets/8823619/16179668/1f65124e-3675-11e6-91f3-d589bbedcc73.jpg)

Подписал действительно неверно. С 6 утра сегодня ботаю, уже немного в прострации.
1 - это [безразмерная величина](https://ru.wikipedia.org/wiki/%D0%91%D0%B5%D0%B7%D1%80%D0%B0%D0%B7%D0%BC%D0%B5%D1%80%D0%BD%D0%B0%D1%8F_%D0%B2%D0%B5%D0%BB%D0%B8%D1%87%D0%B8%D0%BD%D0%B0)

**Comment by timyrka67** [comment]
![17](https://cloud.githubusercontent.com/assets/8823619/16180330/cd3c4576-368a-11e6-803d-c45d7bcc18f0.jpg)
![18](https://cloud.githubusercontent.com/assets/8823619/16180331/cd3f5ee6-368a-11e6-9afe-0a97caac8717.jpg)

Формула которую мы написали неверна. 
Мы не можем умножать какой-то коэфф. на время компьютера. Как мы уже говорили, мы не можем манипулировать временем компьютера. 
А вот умножать время симулятора на разные коэффы можем, получая время проигрывания без учета времени обработки можем. 

 Формулу из второй картинки можно разделить  с двух сторон на дельту т компьютера.
Как я сказал в первом стайде мы считаем изначально дельты симулятора и компа равными, тогда если разделить будут равны 1. 
А произведение двух коэффициентов есть новый общий коэффициент, который мы ошибочно называли скоростью.

Это просто **Коэффициент трансформации**

**Коэффициент трансформации= масштаб*коэффициент скорости**
**коэффициент скорости** - значение на ползунке.

**Единственный нюанс в коде я не умножаю на коэффициент скорости а делю

**Comment by aurusov** [comment]
Звучит норм.

> **Единственный нюанс в коде я не умножаю на коэффициент скорости а делю

Почему ? Где ошибка, на слайдах или в коде ?

**Comment by timyrka67** [comment]
> Почему ? Где ошибка, на слайдах или в коде ?

На слайдах. Хотя по сути небольшая разница. Так как-то понятней.

**Comment by aurusov** [comment]
Ну ok

**Comment by timyrka67** [comment]
Обновил РПЗ.

**Comment by aurusov** [comment]
Посмотрел

**Comment by aurusov** [comment]
Отписался

**Comment by timyrka67** [comment]
Обновил рецензию

**Comment by timyrka67** [comment]
Обновил РПЗ. Нормоконтроль записка прошла. (7% заимствований)
Сегодня фокусируюсь на листах в первую очередь. 
На презентации во вторую.
Рецензию подписал. 

Добавил аннотацию(Комментарии наверху)

**Comment by aurusov** [comment]
Что значит на листах ? Каких таких листах ?

**Comment by aurusov** [comment]
Содержимое аннотации тоже самое ?

**Comment by timyrka67** [comment]
> Что значит на листах ? Каких таких листах ?

Два листа
Диаграмма классов библиотеки моделирования.
Диаграмма классов проигрывателя

Презентация + 2 листа. Такой формат. Они просто будут висеть.
Так можно

**Comment by timyrka67** [comment]
> Содержимое аннотации тоже самое ?

Да. + 2 ФИО. название и подписи

**Comment by aurusov** [comment]
В записке аннотация тоже остается ?

**Comment by timyrka67** [comment]
> В записке аннотация тоже остается ?

Да. А вторая не вшивается. Всего 2.
![fmhlwz_uefq](https://cloud.githubusercontent.com/assets/8823619/16209575/48cdca56-3740-11e6-8b72-01582abd4dec.jpg)

**Comment by aurusov** [comment]
Посмотрел

**Comment by timyrka67** [comment]
Добавил диаграммы классов
Презентацию можно смотреть. Но я еще буду доробатывать.

**Comment by aurusov** [comment]
> Презентацию можно смотреть. Но я еще буду доробатывать.

И вот зачем мне тогда смотреть ? Готов смотреть только то, что уже закончено. Пусть это будет только часть. Напиши какая.

**Comment by timyrka67** [comment]
> И вот зачем мне тогда смотреть ? Готов смотреть только то, что уже закончено. Пусть это будет только часть. Напиши какая.

До слайда 18 можно

**Comment by timyrka67** [comment]
> Зачем тебе эта диаграмма ? Какие класс твои ?

Ради того чтобы показать модел стейт. Пилил это все Паша, но мне тоже нужно. Модел стейт для меня все.

**Comment by aurusov** [comment]
> Ради того чтобы показать модел стейт. Пилил это все Паша, но мне тоже нужно. Модел стейт для меня все.

Неубедительно. Обведи красным классы, которые собираешься использовать в докладе.

**Comment by timyrka67** [comment]
Обновил. РПЗ и листы.

**Comment by aurusov** [comment]
А где архивы ?

**Comment by aurusov** [comment]
1. Титлы битые
2. Титлы надо вставить в начало записки

**Comment by aurusov** [comment]
ping

**Comment by timyrka67** [comment]
Смержил pdf-ки. Добавил исходники.
Открыл доступ на редактирование презентации.

**Comment by aurusov** [comment]
Где исходники листов ?

**Comment by timyrka67** [comment]
Диаграммы классов плеера.cdw + player.uxf
Диаграммы классов симулятора.cdw + simulator.uxf

**Comment by aurusov** [comment]
Выложил, http://raox.ru/?p=1254
